docs/service-mesh/03-02-istio-patch.md

```diff
@@ -3,29 +3,28 @@ title: Istio patch
+1. Istio is installed using the official charts from the currently supported release. The charts reside in the `resources/istio` directory.
 
-2. A custom Istio patch job runs and checks if the detected Istio deployment meets the following criteria:
-  - A specific version of Istio is installed. The required version is defined in the [`values` file](https://github.com/kyma-project/kyma/blob/master/resources/istio-kyma-patch/values.yaml#L11) of the patch.
+2. A custom Istio patch job runs and checks if the detected Istio Deployment meets the following criteria:
+  - A specific version of Istio is installed. The required version resides in the [`values`](https://github.com/kyma-project/kyma/blob/master/resources/istio-kyma-patch/values.yaml#L11) file.
   - Mutual TLS (mTLS) policy is enabled and set to `strict`.
   - [Istio policy enforcement](https://istio.io/docs/tasks/policy-enforcement/enabling-policy/) is enabled. 
   - Automatic sidecar injection is enabled.
   - Istio `policies.authentication.istio.io` CustomResourceDefinition (CRD) is present in the system.
 
-If the Istio deployment fails to meet any of these criteria, the patch fails, which results in a failed installation. If the installation failed, get the Istio patch logs to find out which criteria the Istio deployment didn't' meet. Run: 
+If the Istio Deployment fails to meet any of these criteria, the patch fails, which results in a failed installation. In such a case, get the Istio patch logs to find out which criteria the Istio Deployment didn't meet. Run: 
 
-```
+```bash
 kubectl logs -n istio-system -l app=istio-kyma-patch
 ```
 
-To learn more about the custom Istio patch applied in Kyma, see the `components/istio-kyma-patch/` directory.
-
->**NOTE:** Istio patch is an optional component and can be disabled. However, it is enabled by default. Read [this](/root/kyma/#configuration-custom-component-installation) to learn how to disable components. 
+>**NOTE:** The Istio patch component is enabled by default, but you can disable it at any time using [these](/root/kyma/#configuration-custom-component-installation) instructions. 
 
 ## Use an existing Istio installation with Kyma
 
-You can use an existing installation of Istio running on Kubernetes with Kyma. In such a case is required to enable the patch component to verify if all the required options are set. 
+You can use an existing installation of Istio with your Kyma installation. Perform the following steps:
 
-To allow such implementation, you must install Kyma without Istio. Read [this](/root/kyma/#configuration-custom-component-installation) document for more details.
+1. Make sure the Istio Patch component is enabled and meets the criteria required by the Istio patch job.
```

Comment:
Istio Patch component IS a job in fact and it checks if Istio meets certain criteria. Here it sounds like the patch checks itself.

Reply:
I don't really think this is worthy of creating an ordered list. It's two things really - 
- have your Istio running in the cluster
- install Kyma with Istio disabled, don't disable Istio Patch (the patch is enabled by default)